File: app/src/main/java/com/startogamu/zikobot/core/module/spotify_api/model/SpotifyTrack.java
package com.startogamu.zikobot.core.module.spotify_api.model;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * Created by josh on 29/03/16.
 */
@Data
@AllArgsConstructor(suppressConstructorProperties = true)
public class SpotifyTrack {
    @SerializedName("id")
    private String id;
    @SerializedName("name")
    private String name;
    @SerializedName("album")
    private SpotifyAlbum album;
    @SerializedName("artists")
    private ArrayList<SpotifyArtist> artists;
    @SerializedName("type")
    private String type;
    @SerializedName("duration_ms")
    private long duration;


}

File: data/src/main/java/com/joxad/zikobot/data/module/spotify_api/model/SpotifyTrack.kt
package com.joxad.zikobot.data.module.spotify_api.model

import com.google.gson.annotations.SerializedName
import java.util.*

/**
 * Created by josh on 29/03/16.
 */
class SpotifyTrack {
    @SerializedName("id")
    val id: String? = null
    @SerializedName("name")
    val name: String? = null
    @SerializedName("album")
    val album: SpotifyAlbum? = null
    @SerializedName("artists")
    val artists: ArrayList<SpotifyArtist>? = null
    @SerializedName("type")
    val type: String? = null
    @SerializedName("duration_ms")
    val duration: Long = 0


}
